getting-started/go.md

```diff
@@ -1 +1,112 @@
-TODO
+# Getting Started with IPLD in Go
+
+In Go, the primary way to use IPLD is via the `github.com/ipld/go-ipld-prime`
+module.
+
+Its API is centered around `Node`s, which can be built from scratch and encoded
+into blocks, where we can then obtain `Link`s to them.
+
+First, we'll need to import some IPLD packages and define a static link builder:
```

Comment:
"static" sounds a little funny to me here.

How about "define a link builder, which specifies all the codecs and hashing features we'll use"?

Reply:
sure, done